Moving config to a new directory

mys721tx opened the pull request:
Moves config to a new directory. In this way, the directory can be
mounted as a volume in docker and the docker image will not embedded any
configuration files.

mys721tx committed:
Moving config to a new directory

Moves config to a new directory. In this way, the directory can be
mounted as a volume in docker and the docker image will not embedded any
configuration files.

The configuration file is also changed to JSON in order to prevent
arbitrary code execution.

dotkrnl commented:
tuna/fishroom 项目由于授权条款原因，部分贡献者维护能力受限，已通过 #91 以及 416726576e51016a77a4eb2996db2321d3cae133 将所用授权条款修改为 GPLv3。

由于此 pull request 中代码的授权方式仍为 AGPL，未来的维护者暂无法对其进行 merge 操作。如您仍希望代码接受合并，请回复同意已将您这一贡献的授权方式修改为 GPLv3。

谢谢！

mys721tx commented:
同意

dependabot-preview[bot] committed:
Bump tornado from 4.5.2 to 6.0.3

Bumps [tornado](https://github.com/tornadoweb/tornado) from 4.5.2 to 6.0.3.
- [Release notes](https://github.com/tornadoweb/tornado/releases)
- [Changelog](https://github.com/tornadoweb/tornado/blob/master/docs/releases.rst)
- [Commits](https://github.com/tornadoweb/tornado/compare/v4.5.2...v6.0.3)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
Bump aiohttp from 2.3.3 to 3.5.4

Bumps [aiohttp](https://github.com/aio-libs/aiohttp) from 2.3.3 to 3.5.4.
- [Release notes](https://github.com/aio-libs/aiohttp/releases)
- [Changelog](https://github.com/aio-libs/aiohttp/blob/master/HISTORY.rst)
- [Commits](https://github.com/aio-libs/aiohttp/compare/v2.3.3...v3.5.4)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
[Security] Bump marshmallow from 2.1.0 to 2.15.1

Bumps [marshmallow](https://github.com/marshmallow-code/marshmallow) from 2.1.0 to 2.15.1. **This update includes a security fix.**
- [Release notes](https://github.com/marshmallow-code/marshmallow/releases)
- [Changelog](https://github.com/marshmallow-code/marshmallow/blob/dev/CHANGELOG.rst)
- [Commits](https://github.com/marshmallow-code/marshmallow/compare/2.1.0...2.15.1)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>
